coda-src/venus/fso.h

```diff
@@ -344,6 +344,8 @@ class fsobj {
   friend void RecoverPathName(char *, VenusFid *, ClientModifyLog *, cmlent *);
 
     int MagicNumber;
+    
+    CacheChunkList active_segments; /**< List of active cache file segments */
```

Comment:
Is this transient or recoverable? i.e. does this list of active segments survive a client restart? Are all updates to this list persisted using RVM transactions and if it is transient, are we resetting the state during startup.

Reply:
It's transient. I'm adding `/*T*/`.